exercise/exercise_11.py

```diff
@@ -14,4 +14,27 @@
 
 # Как мы видим, 28 - первое треугольное число, у которого более пяти делителей.
 
-# Каково первое треугольное число, у которого более пятисот делителей?
+# Каково первое треугольное число, у которого более десити делителей?
+
+
+def exercise_11(a):
+
+    n = 1
+    summa = 0
+    b = []
+    c = True
+    while c:
+        k = 0
```

Comment:
You define k as 0 here, but lower u redefine it in the cycle, so, here is question, is it need here?